WalletWasabi.Fluent/Controls/ContentArea.axaml

```diff
@@ -70,7 +70,10 @@
                                             Classes="plain"
                                             Command="{Binding BackCommand}"
                                             IsVisible="{TemplateBinding EnableBack}">
-                                        <PathIcon Data="{StaticResource arrow_left_regular}" Width="16" />
+                                      <i:Interaction.Behaviors>
+                                        <behaviors:ButtonExecuteCommandOnKeyDownBehavior Key="Back"/>
```

Comment:
Should support `IsEnable` and bind it to `IsActive`. Because only the topmost (IsActive) dialog should listen to this keypress.

We do the same as the continue button's `IsDefault` property.

Reply:
Added